### PR TITLE
testing/wabt: new aport

### PR DESCRIPTION
**Pull request by ids1024**

### CODE REVIEW

File: testing/wabt/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Contributor: Ian Douglas Scott <ian@iandouglasscott.com>
+# Maintainer: Ian Douglas Scott <ian@iandouglasscott.com>
+pkgname=wabt
+pkgver=1.0.11
+pkgrel=0
+pkgdesc="WebAssembly binary toolkit"
+arch="all"
+url="https://github.com/WebAssembly/wabt"
+license="Apache-2.0"
+makedepends=cmake
```
Comment (maxice8) [inline testing/wabt/APKBUILD]: makedepends should be quoted

File: testing/wabt/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::$url/archive/$pkgver.tar.gz"
+
+prepare() {
+	default_prepare
```
Comment (maxice8) [inline testing/wabt/APKBUILD]: prepare() is defined as `default_prepare` by default, so this whole definition can be removed.

**Comment by ids1024** [comment]
The tests inconveniently use vendored dependencies. I've raised an issue about that upstream: https://github.com/WebAssembly/wabt/issues/1103.

Probably including a patch to use the packaged versions would be a good short-term solution, until it is (hopefully) addressed upstream. But I'll see what response that issue gets.

**Comment by maxice8** [comment]
You can set !check in options="" with a comment in the mean time

**Comment by ids1024** [comment]
I've amended the commit to address these concerns.

**Pull request closed without merging**